bin/install-dependencies.bash

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Installs dependency for Ubuntu Linux, which is used on Github Actions
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Print functions
+stdmsg() {
+  local IFS=' '
+  printf '%s\n' "$*"
+}
+
+errmsg() {
+  stdmsg "$*" 1>&2
+}
+
+# Trap exit handler
+trap_exit() {
+  # It is critical that the first line capture the exit code. Nothing
+  # else can come before this. The exit code recorded here comes from
+  # the command that caused the script to exit.
+  local exit_status="$?"
+
+  if [[ ${exit_status} -ne 0 ]]; then
+    errmsg 'There is an error installing the dependencies.'
+    exit 1
+  fi
+}
+trap trap_exit EXIT
+
+# Returns 0 if the environment variable is set to any value including
+# null, 1 otherwise.
+is_set() {
+  [[ -n ${!1+x} ]]
+}
+
+# Update package list
+apt-get update
+
+# Install wget and shellcheck
+apt-get update
+apt-get install wget xz-utils -y
+
+# Install shellcheck
+wget https://github.com/koalaman/shellcheck/releases/download/v0.10.0/shellcheck-v0.10.0.linux.aarch64.tar.xz --output-document /tmp/shellcheck.tar.xz
+tar xJf /tmp/shellcheck.tar.xz --directory /tmp
```

Comment:
Good use of the `--directory` option! But, note well what the manpage says:

```
       -C, --directory=DIR
              Change to DIR before performing any operations.  **This option is order-sensitive**, i.e. it affects all options that follow.
```

Also, [the version of `tar` shipped with MacOS](https://ss64.com/mac/tar.html) (which is based on BSD tar, not GNU tar) only knows about `-C`, not `--directory`, so it's often safer to use `-C` here.

So, in order to make sure that all tar commands use the right working directory, and the script has a fighting chance of working cross-platform, write it like this:

```
tar -C "${temp_download_dir}" -xJf shellcheck.tar.xz
```